itou/templates/apply/list_of_available_exports.html

```diff
@@ -4,20 +4,43 @@
 
 {% block title %}Export des candidatures {{ block.super }}{% endblock %}
 
-{% block breadcrumbs %}
+{% block title_prevstep %}
     {% include "layout/previous_step.html" with back_url=back_url only %}
 {% endblock %}
 
-{% block content_title %}
-    <h1>
-        Exporter les candidatures
-        {% if export_for == "siae" %}
-            reçues
-        {% else %}
-            envoyées
-        {% endif %}
-    </h1>
-    {% if export_for == "siae" %}<h2>{{ siae.display_name }}</h2>{% endif %}
+{% block title_content %}
+    <div class="d-flex flex-column flex-md-row gap-3 mb-3 justify-content-md-between">
+        <div>
+            <h1>
+                Exporter les candidatures
+                {% if export_for == "siae" %}
+                    reçues
+                {% else %}
+                    envoyées
+                {% endif %}
+            </h1>
+            {% if export_for == "siae" %}<h2>{{ siae.display_name }}</h2>{% endif %}
+        </div>
+        <div>
+            {% if export_for == "siae" %}
+                <a class="btn btn-lg btn-primary btn-ico w-100 w-md-auto"
+                   {% matomo_event "candidature" "exports" "export-siae" %}
+                   aria-label="Télécharger cet export SIAE"
+                   href="{% url 'apply:list_for_siae_exports_download' %}">
+                    <i class="ri-download-line font-weight-medium" aria-hidden="true"></i>
+                    <span>Télécharger toutes les candidatures</span>
+                </a>
+            {% else %}
+                <a class="btn btn-lg btn-primary btn-ico w-100 w-md-auto"
+                   {% matomo_event "candidature" "exports" "export-prescripteur" %}
+                   aria-label="Télécharger cet export prescripteur"
+                   href="{% url 'apply:list_prescriptions_exports_download' %}">
+                    <i class="ri-download-line font-weight-medium" aria-hidden="true"></i>
+                    <span>Télécharger toutes les candidatures</span>
+                </a>
+            {% endif %}
```

Comment:
```suggestion
                <a class="btn btn-lg btn-primary btn-ico w-100 w-md-auto"
                   {% if export_for == "siae" %}
                       {% matomo_event "candidature" "exports" "export-siae" %}
                       aria-label="Télécharger cet export SIAE"
                       href="{% url 'apply:list_for_siae_exports_download' %}">
                   {% else %}
                       {% matomo_event "candidature" "exports" "export-prescripteur" %}
                       aria-label="Télécharger cet export prescripteur"
                       href="{% url 'apply:list_prescriptions_exports_download' %}">
                    {% endif %}
                    <i class="ri-download-line font-weight-medium" aria-hidden="true"></i>
                    <span>Télécharger toutes les candidatures</span>
                </a>
```
Je pensais éventuellement à cela

Reply:
Ah ouais carrement. Donc ça serait a faire [là](https://github.com/gip-inclusion/les-emplois/blob/0db5c3e37b3e4a4c507594a3fa4405963c60e221/itou/templates/apply/list_of_available_exports.html#L94) et [là aussi](https://github.com/gip-inclusion/les-emplois/blob/0db5c3e37b3e4a4c507594a3fa4405963c60e221/itou/templates/apply/list_of_available_exports.html#L112)

Reply:
Déjà, le make fix aime pas trop. Du coup, Il mets le `<a>` avec attibuts et condtions sur une seule ligne. 
Niveau lisibilité du code, c'est pas terrible :/

Reply:
Alors restons sur ta version :)